soap-connect/integration-tests/pom.xml

```diff
@@ -14,6 +14,7 @@
     <packaging>jar</packaging>
 
     <properties>
+        <cxfVersion>3.0.13</cxfVersion>
```

Comment:
Because it was using this one: https://github.com/mulesoft/mule/pull/5040/files#diff-600376dffeb79835ede4a0b285078036L159.